library/azure_rm_cosmosdbaccount.py

```diff
@@ -403,12 +422,13 @@ def format_item(self, d):
         return d
 
 
-def default_compare(new, old, path):
+def default_compare(new, old, path, exceptions):
```

Comment:
the interface is even more harder to use :)

Reply:
well.... at least it's smaller ;-)
I will add documentation...